src/PathResolver.mad

```diff
@@ -140,8 +152,12 @@ export processPath = (state) => pipe(
     isItemView(state.path),
     pipe(
       List.first,
-      where { Just(m) => findItem(state.path, m) }
+      map(findItem(state.path)),
+      where {
+        Just(m) =>
+          findItem(state.path, m)
```

Comment:
This one is the only piece which seems like it could be incorrect still -- getting an incomplete warning for this.

I was thinking either I'd address in the morning or do a beta release and test downstream.